src/assets/actions/actionsIcons.js

```diff
@@ -1,7 +0,0 @@
-const actionIcons = {
```

Comment:
This file is used to fill the icons for the action menu, maybe simply change it's location?

Reply:
no I check, we have two files, `actionIcons.js` and `actionsIcons.js`, only `actionIcons.js` is used.
In your last pr, you forget to delete one of them (And I don't see it in review) ;)